libraries/AP_RCTelemetry/AP_Spektrum_Telem.cpp

```diff
@@ -435,7 +435,7 @@ void AP_Spektrum_Telem::calc_airspeed()
     AP_AHRS &ahrs = AP::ahrs();
     WITH_SEMAPHORE(ahrs.get_semaphore());
 
-    const AP_Airspeed *airspeed = ahrs.get_airspeed();
+    const AP_Airspeed *airspeed = AP::airspeed();
     float speed = 0.0f;
     if (airspeed && airspeed->healthy()) {
```

Comment:
needs explicit nullptr chk instead of nonzero

Reply:
As elsewhere, unrelated to my PR.  Not a bad change, just unrelated.